### ISSUE TITLE
Composite key break multi row results

### ISSUE THREAD

**Issue by lthi747** [wsporto/typesql#37]
It seems that if using composite key, the result of function is forced to first row

```sql
CREATE TABLE t1 (
	u1_id INTEGER,
	u2_id INTEGER,
	name TEXT,
	PRIMARY KEY (u1_id, u2_id)
);
```

```sql
-- get-all-u1.sql
select * from t1 where u1_id = :u1_id limit 100
```

**Comment by wsporto** [comment]
Thanks for reporting, I'll look into that. 

In the meantime, to avoid getting stuck on this error, you can try the following:

1. Modify the query slightly to ensure TypeSQL generates multiple rows. For example, use `SELECT * FROM t1 WHERE u1_id > :u1_id LIMIT 100` (using > instead of =).
2. Let TypeSQL generate the boilerplate code.
3. Delete the .sql file or rename it to .txt (for example) to avoid regenerating the code.
4. Update the generated code with the correct query.

**Comment by wsporto** [comment]
This should be fixed in version v0.14.3.
